app/services/db/sp_return_log.rb

```diff
@@ -1,44 +1,18 @@
 module Db
   class SpReturnLog
-    # rubocop:disable Rails/SkipsModelValidations
-    def self.update_user(request_id:, user_id:)
-      return if request_id.blank?
-      ::SpReturnLog.where(request_id: request_id).update_all(user_id: user_id)
-      nil
-    end
-
-    def self.add_return(request_id:, user_id:, billable:, ial:)
-      ::SpReturnLog.where(request_id: request_id).update_all(
-        user_id: user_id,
-        returned_at: Time.zone.now,
-        billable: billable,
-        ial: ial,
-      )
-      nil
-    end
-    # rubocop:enable Rails/SkipsModelValidations
-
-    def self.create_request(request_id:, ial:, issuer:)
+    def self.create_return(request_id:, user_id:, billable:, ial:, issuer:)
+      now = Time.zone.now
       ::SpReturnLog.create!(
         request_id: request_id,
+        user_id: user_id,
+        billable: billable,
         ial: ial,
         issuer: issuer,
-        requested_at: Time.zone.now,
+        returned_at: now,
+        requested_at: now,
```

Comment:
could we pull requested_at from session?

Reply:
ah, good point, we should be able to!